src/Illuminate/Mail/Transport/SparkPostTransport.php

```diff
@@ -54,7 +54,9 @@ public function send(Swift_Mime_SimpleMessage $message, &$failedRecipients = nul
 
         $message->setBcc([]);
 
-        $response = $this->client->post('https://api.sparkpost.com/api/v1/transmissions', [
+        $endpoint = $this->getOptions()['endpoint'] ?? 'https://api.sparkpost.com/api/v1/transmissions';
```

Comment:
Don't you think it would be a breaking change if `endpoint` is not set in service config? https://github.com/laravel/laravel/blob/master/config/services.php#L29

Reply:
I did forget about that detail on my first attempt at solving this, but this approach should prevent any errors from undefined configurations. 
If someone were to backport this to an older version of laravel that doesn't require PHP 7, they'd need to manually check the options. Here's a little 3v4l snippet: https://3v4l.org/No8Mm

Reply:
I believe it should be set in service config just for the sake of making it obvious that it's customizable (which would mean moving the default endpoint from the transport class to the config array). It would save time for some developers.